src/test/config.sol

```diff
@@ -126,7 +126,7 @@ contract Config {
             osm_mom_authority:     chief,                   // OsmMom authority
             flipper_mom_authority: chief,                   // FlipperMom authority
             clipper_mom_authority: chief,                   // ClipperMom authority
```

Comment:
d3m mom authority check could be added here

Reply:
added